sunpy/map/sources/soho.py

```diff
@@ -148,6 +148,10 @@ def nickname(self):
         filter = self.meta.get('filter', '')
         return f'{self.instrument}-{self.detector} {filter}'
 
+    @nickname.setter
+    def nickname(self, value):
+        raise RuntimeError("Cannot manually set nickname for LASCOMap")
```

Comment:
Do we have this for other sources?

Reply:
This is because GenericMap defines a setter, but LASCOMap doesn't respect that and instead just generates the nickname from the metadata. Perhaps I should put this in a separate PR anyway since it's more of a bugfix.